modules/backup-and-restore/pages/point-in-time-restore.adoc

```diff
@@ -0,0 +1,76 @@
+
+The point-in-time restore is based on full backup and differential backup. So we need to ensure the time point to restore is not under these cases:
+
+
+* the time points before the first full backup or after the current time.
+* the time points before the last restore and aren’t covered by existing backups. Here not covered means the time points are not contained by any backup’s time points range [POINT-IN-TIME START TIME,  CREATED AT]. We can check the backup’s time points range by `gadmin backup list`. here is an example:
+
+[console, gsql]
+---
+$ gadmin backup list
+|================================================================================================================================================
+|             TAG              |    TYPE     | VERSION |  SIZE  |             BASE             | POINT-IN-TIME START TIME |      CREATED AT      
+| backup-2024-06-13T061939.131 | FULL        | 4.1.0   | 1.5 MB |                              | 2024-06-13T06:19:39Z     | 2024-06-13T06:19:39Z 
+| backup-2024-06-13T063915.054 | INCREMENTAL | 4.1.0   | 1.5 MB | backup-2024-06-13T061939.131 | 2024-06-13T06:19:39Z     | 2024-06-13T06:39:15Z 
+| backup-2024-06-13T081939.120 | FULL        | 4.1.0   | 1.5 MB |                              | 2024-06-13T08:19:39Z     | 2024-06-13T08:19:39Z 
+|================================================================================================================================================
+---
+
+here, the time range for backup `backup-2024-06-13T061939.131` is `[2024-06-13T06:19:39Z,2024-06-13T06:19:39Z]`, therefore, for a full backup is just a point. As for the incremental backup `backup-2024-06-13T063915.054`, its range is `[2024-06-13T06:19:39Z , 2024-06-13T06:39:15Z]`. so, for instance, if we have performed a restore at `2024-06-14T06:19:39Z`. And then we can’t restore to the timepoint `2024-06-13T07:00:00Z`, for it is not in any backup's time range. however, `2024-06-13T06:35:00Z` is supported for it is covered by the time range `[2024-06-13T06:19:39Z , 2024-06-13T06:39:15Z]` of the backup `backup-2024-06-13T063915.054`.
+
+
+[NOTE]
+====
+because the restore action will make the uncovered timepoint unavailable to restore, if you are not very sure about the time point to restore,  we’d better ensure the possible time points are all covered by existing backups. if there is not this kind of backup, you may create a by `gadmin backup create <backup_tag> --incremental --base <full_backup_tag>`, the new backup will cover the range from the full backup <full_backup_tag>'s create time to the current time.
+====
+
+== Key Benefits
+
+The primary advantage of Point-in-Time Restore is the flexibility it offers in selecting any specific time point on the timeline for restoration.
```

Comment:
Let's expand the key benefits:
The primary advantage of Point-in-Time Restore is the flexibility it offers, allowing users to restore the database to a specific time point. This feature is particularly useful in scenarios where unwanted changes or data corruption need to be undone without losing other valuable work done after the last backup.